src/api/seancesService.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+import type { Seance } from '@/types/seance'
+
+axios.defaults.baseURL = 'http://localhost:3000';
+
+export const getAllSeance = async (): Promise<Seance[]> => {
+  const response = await axios.get('/seances') 
+  return response.data
+}
+
+export const getDaySeance = async (date: string): Promise<Seance[]> => {
+  const response = await axios.get(`/seances?date=${date}`) 
```

Comment:
Depending on date format you might want to encode the value to get rid of some special characters. 

In general it's a good idea to do that for all query parameters either with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent or a lib https://www.npmjs.com/package/qs which is part of vue 3 dependencies anyway

Reply:
date before getting here is always format to same specification of `toISOString().substring(0, 10)`

Reply:
That's fine 👍